include/ignition/gui/qml/IgnCard.qml

```diff
@@ -39,6 +39,11 @@ Pane {
    */
   property bool showCloseButton: true
 
+  /**
+   * True to have a minimize button
+   */
+  property bool showMinimizeButton: true
```

Comment:
How about calling it `showCollapseButton`? Just so the API is consistent. Users are able to set this, as well as the state, from config files.